src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -370,11 +391,55 @@ export const PublishingAccordion = ({
               variant="outlined"
               data-testid={dataTestId.registrationLandingPage.tasksPanel.publishingRequestRejectButton}
               startIcon={<CloseIcon />}
-              onClick={() => ticketMutation.mutate({ status: 'Closed' })}
+              onClick={() => setOpenRejectionDialog(true)}
               loading={isLoading === LoadingState.RejectPublishingRequest}
               disabled={isLoadingData || isLoading !== LoadingState.None}>
               {t('registration.public_page.reject_publish_request')} ({filesAwaitingApproval})
             </LoadingButton>
+
+            <Dialog open={openRejectionDialog} onClose={() => setOpenRejectionDialog(false)}>
+              <DialogTitle fontWeight="bold">{t('registration.public_page.reject_publish_request')}</DialogTitle>
+              <DialogContent>
+                <Trans
+                  i18nKey="registration.public_page.reject_publish_request_description"
+                  components={[<Typography paragraph />]}
+                />
+                <TextField
+                  data-testid={dataTestId.registrationLandingPage.tasksPanel.publishingRequestRejectionMessageTextField}
+                  inputProps={{ maxLength: 160 }}
+                  variant="filled"
+                  multiline
+                  minRows={3}
+                  maxRows={Infinity}
+                  fullWidth
+                  required
+                  label={t('registration.public_page.reason_for_rejection')}
+                  FormHelperTextProps={{ sx: { textAlign: 'end' } }}
+                  helperText={`${rejectionReason.length}/160`}
+                  value={rejectionReason}
+                  onChange={(event) => setRejectionReason(event.target.value)}
+                />
+              </DialogContent>
+              <DialogActions>
+                <Button
+                  data-testid={dataTestId.registrationLandingPage.tasksPanel.cancelRejectionButton}
+                  onClick={() => setOpenRejectionDialog(false)}>
+                  {t('common.cancel')}
+                </Button>
+                <LoadingButton
+                  data-testid={dataTestId.registrationLandingPage.tasksPanel.rejectionDialogConfirmButton}
+                  disabled={!rejectionReason}
+                  loading={isLoading === LoadingState.RejectPublishingRequest}
+                  variant="contained"
+                  onClick={() =>
+                    handleRejectPublishFileRequest(
+                      `${t('registration.public_page.reason_for_rejection')}: ${rejectionReason}`
+                    )
```

Comment:
Litt usikker på om jeg skla ha med prefix her eller ikke. Det er ingen andre måter å identifisere at det er en begrunnelse for avvisning (bortsett fra context), men den vil dog variere basert på hvilket språk brukeren har valgt i applikasjonen, ikke språket på innholdet. Hva tenker dere?

Reply:
Usikker. Synes det er litt mystisk at den plutselig dukker opp etter at meldingen er sendt. Bruker kan f eks ha innledet meldingen med ish samme tekst, og da blir det dobbelt opp. Hva om man heller prefiller input-boksen med denne teksten, og setter på en validator som sjekker at bruker har skrevet mere/noe annet?